src/rules/noAnyRule.ts

```diff
@@ -42,7 +43,10 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class NoAnyWalker extends Lint.RuleWalker {
     public visitAnyKeyword(node: ts.Node) {
-        this.addFailureAtNode(node, Rule.FAILURE_STRING);
+        const fix = this.createFix(
+            this.createReplacement(node.getStart(), node.getWidth(), "{}"),
```

Comment:
`any` (both a supertype and a subtype of every type) is semantically different from `{}` (a supertype of every type). Consider the following example:

```
declare const a1: any;
const a2: number = a1;  // `any` can be cast to `number`

declare const b1: {};
const b2: number = b1;  // error: `{}` cannot be cast to `any`
```

It is okay to advice using `{}` in linter warning message. But autofix should try to preserve the original semantic as much as possible without changing the semantic of the source code. At the minimum least, can `tslint` provide option to selectively disable autofix for this specific linter rule?